### ISSUE TITLE
Visual Studio Community 2013 Compiler Error C2064

### ISSUE THREAD

**Issue by hezhao** [pizthewiz/Cinder-EDSDK#4]
First, in `Camera.h` I had to add the following system header files to make `std::function` and  `std::shared_ptr`  work in Visual Studio Community 2013.
 ```
#include <functional> 
#include <memory>
``` 
After that, I received a number of [C2064 errors](https://msdn.microsoft.com/en-us/library/z72c1dfd.aspx), couldn't quite figure out a solution but I think it's related to the handlers `mAddedHandler`, `mRemovedHandler`, `mEnumeratedHandler` with type `std::function`.

`error C2064: term does not evaluate to a function taking 1 arguments	C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\include\xrefwrap	283`

**Comment by pizthewiz** [comment]
This is when using `Camera.h` without the Cinder includes?
```C++
#include "cinder/Cinder.h"
#include "cinder/ImageIo.h"
```

**Comment by hezhao** [comment]
Yes, sorry I forgot to mention, I removed Cinder headers, precisely I'm using [EDSDK-cpp](https://github.com/hezhao/EDSDK-cpp).

**Comment by pizthewiz** [comment]
Hmm, were you hoping that Cinder-EDSDK would support Cinder-less use? It might make sense in a branch but I think master could safely assume Cinder, no?
